packages/core-mobile/android/app/src/main/java/com/avaxwallet/CustomGlideModule.kt

```diff
@@ -0,0 +1,7 @@
+package com.avaxwallet
+
+import com.bumptech.glide.annotation.GlideModule
+import com.bumptech.glide.module.AppGlideModule
+
+@GlideModule
+class CustomGlideModule : AppGlideModule() {}
```

Comment:
If we don't create this module, expo-image fails to load images. ([source](https://stackoverflow.com/questions/49901629/glide-showing-error-failed-to-find-generatedappglidemodule))